robot/markers.py

```diff
@@ -133,3 +171,14 @@ def cartesian(self) -> CartCoord:
         The camera's position is the origin of the co-ordinate space.
         """
         return CartCoord(*self._raw_data['cartesian'])
+
+    @property
+    def spherical(self) -> SphericalCoord:
+        """
+        The position of the marker in Spherical co-ordinates.
+
+        This co-ordinate space describes a point as a angles about the Cartesian
+        x and y axes and a distance from the camera. Note: this co-ordinate
+        space is differnt to the usual representation of a spherical space.
```

Comment:
differnt -> different

Reply:
Perhaps its worth briefly clarifying what the "usual representation" is, and in what way our library differs from it?

Reply:
c944d4f (fixes typo)

Reply:
3676b76 does this in the type docstring; I'd rather keep the property docstring short and related to the property rather than duplicating the type docstring.